README.md

```diff
@@ -1,2 +1,2 @@
 # terraform-in-21-days
-my 1st commit !
+my 1st commit !brew update
```

Comment:
No need to update README.md anymore. In classes it was just done for the demo purposes

Reply:
Noted !